modules/integration_aws-efs/conf/03-efs-iopsread.yaml

```diff
@@ -0,0 +1,21 @@
+module: "AWS EFS"
+name: "IOPs Read Stats"
+filtering: "filter('namespace', 'AWS/EFS')"
+value_unit: "IOPs"
+transformation: ".max(over='15m')"
+signals:
+  data_read_io_bytes:
+    metric: "DataReadIOBytes"
+    filter: "filter('stat', 'mean')"
+  meta_data_io_bytes:
+    metric: "MetadataIOBytes"
+    filter: "filter('stat', 'mean')"
+  signal:
+    formula:
+      (meta_data_io_bytes/data_read_io_bytes).scale(100)
```

Comment:
I don't understand this one. I don't know efs but reading the cloudwatch metrics should we not monitor `MeteredIOBytes/PermittedThroughput*100` instead.

if yes, the "write" version of this detector could be deleted. If you want may be it can be replaced by the `BurstCreditBalance` which could useful to monitor.

Reply:
this doc is awesome and give all necessary formulas to monitor throughput related metrics https://docs.aws.amazon.com/efs/latest/ug/monitoring-metric-math.html

Reply:
I make some changes, could you check if it is ok for you @nicolasvion ?